core/src/main/scala/com/databricks/labs/remorph/intermediate/adf/activities/Activity.scala

```diff
@@ -0,0 +1,29 @@
+package com.databricks.labs.remorph.intermediate.adf.activities
+
+import com.databricks.labs.remorph.intermediate.adf.PipelineNode
+import com.databricks.labs.remorph.intermediate.adf.services.LinkedServiceReference
+
+case class Activity(
+    dependsOn: Seq[ActivityDependency],
+    description: Option[String],
+    linkedServiceName: Option[LinkedServiceReference],
+    name: Option[String],
+    onInactiveMarkAs: Option[OnInactiveMarkAs],
+    policy: Option[ActivityPolicy],
+    state: Option[ActivityState],
+    activityType: Option[String],
+    activityProperties: Option[ActivityProperties],
+    userProperties: Seq[UserProperty])
+    extends PipelineNode {
+  override def children: Seq[PipelineNode] = Seq() ++ dependsOn ++ linkedServiceName ++
+    policy ++ userProperties
+}
```

Comment:
If we are sure that `linkedServiceName`, `policy` and `userProperties` are optional

```suggestion
  override def children: Seq[PipelineNode] = dependsOn ++ linkedServiceName.toSeq ++
    policy.toSeq ++ userProperties.toSeq
}
```

otherwise, we should go for 
```scala
override def children: Seq[PipelineNode] = dependsOn ++ Seq(linkedServiceName, policy, userProperties)
```

Reply:
This is done. I made fields required and used the last pattern.